scripts/oracle/abstain_oracle_voting.py

```diff
@@ -0,0 +1,85 @@
+import time
+import subprocess
+import requests
+from pathlib import Path
+import sys
+
+# for internal loadtest only
+CMD_TMPL = "printf '{password}\n' | " + str(Path.home()) + "/go/bin/seid"
+PREVOTE_TMPL = (
+    " tx oracle aggregate-prevote abc 0uusdc,0uatom {val_addr} --from={key} "
+    "--chain-id={chain_id} --fees=2000usei -y --broadcast-mode=sync"
+)
+VOTE_TMPL = (
+    " tx oracle aggregate-vote abc 0uusdc,0uatom {val_addr} --from={key} "
+    "--chain-id={chain_id} --fees=2000usei -y --broadcast-mode=sync"
+)
+
+KEY = "default_account"
+CHAIN_ID = "sei-internal-testnet"
```

Comment:
should we change this to sei-testnet-2 just so folks don't mess it up and ask questions

Reply:
will do